src/routes/courses/handlers.ts

```diff
@@ -28,6 +32,89 @@ export async function allCourses(req: Request, res: Response) {
   }
 }
 
+export async function getCourseById(req: Request, res: Response) {
+  try {
+    const { id } = req.params;
+    const course = await getById(Number(id));
+    res.json(course);
+  } catch (err) {
+    await handleErrors(err, req, res, logContext);
+  }
+}
+
+export async function updateCourseById(req: Request, res: Response) {
+  try {
+    const { id } = req.params;
+    const userId = await currentUserId(req, res);
+    const user = await userById(userId);
+    const authorization = new UserPolicy(user);
+
+    if (!authorization.isAdmin()) {
+      res.status(403).send('Forbidden');
+      return;
+    }
+
+    const course = await getById(Number(id));
+    if (!course) {
+      res.status(404).send('Course not found');
+      return;
+    }
+    const updated = await course.update(req.body);
```

Comment:
I see there is a mapsTo column on the Courses table. I guess we aren't using it/following that pattern? 
Not a leading question, just curious.

Reply:
Thanks for pointing this out. Using it now: 9f4ebfbd6